### PR TITLE
ON HOLD TO MERGE Update copy for FAQs

### PR DESCRIPTION
**Pull request by sanlouise**
## Description
For this page: https://staging.va.gov/profile/account-security

Replace what's in the blue box currently with:

Have questions about signing in to VA.gov?
Get answers to frequently asked questions about how to sign in, common issues with verifying your identity, and your privacy and security on VA.gov.

Go to FAQs about these topics:
Signing in to VA.gov
Verifying your identity on VA.gov
Privacy and security on VA.gov

For this page: https://staging.va.gov/profile/connected-applications

Replace what's in the blue box currently with:

Have more questions about connected apps?

## Testing done
Looks good locally, can't test URLs till the pages are live.

## Screenshots

image

image

## Acceptance criteria
- [x] Update copy in connected apps and account security pages
- [x] Remove redundant FAQ links in connected apps

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile/components/account-security/AccountSecurityContent.jsx [hunk 1]
```diff
@@ -89,9 +89,12 @@ export const AccountSecurityContent = ({
             security on VA.gov.
           </p>
 
-          <h4>Go to FAQs about these topics:</h4>
+          <h4 className="vads-u-margin-top--1">
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/account-security/AccountSecurityContent.jsx:92]: Are these margin overrides really required? If they don't agree with the mocks I'd think that whatever our design system does by default is the better source of what's correct.

File: src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx [hunk 2]
```diff
@@ -165,9 +165,10 @@ export class ConnectedApps extends Component {
                   'profile-section': 'vets-faqs',
                 })
               }
-              href="/sign-in-faq/#connecting-third-party-non-va-"
+              href="/resources/connecting-third-party-apps-to-your-va-gov-profile/"
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx:168]: this-is-a-very-long-url

File: src/applications/personalization/profile/tests/components/account-security/AccountSecurityContent.unit.spec.jsx [hunk 3]
```diff
@@ -115,18 +115,23 @@ describe('AccountSecurityContent', () => {
     wrapper = shallow(<AccountSecurityContent {...makeDefaultProps()} />);
     const alertBox = wrapper.childAt(1);
     const alertBoxText = alertBox.find('p');
-    const alertBoxLink = alertBox.find('a');
+    const firstAlertBoxLink = alertBox.find('a').first();
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/components/account-security/AccountSecurityContent.unit.spec.jsx:118]: not saying you should change this, but I have really grown to hate enzyme tests like this! 😆

**Review by erikphansen — APPROVED** [review]
Just a question about the margin overrides. But maybe making the container a flex container changed the spacing, requiring the overrides? 🤷

File: src/applications/personalization/profile/tests/components/account-security/AccountSecurityContent.unit.spec.jsx (hunk 3 above)
Comment (sanlouise) [reply src/applications/personalization/profile/tests/components/account-security/AccountSecurityContent.unit.spec.jsx:118]: Yeah, they are pretty ugly! 💀

File: src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx (hunk 2 above)
Comment (sanlouise) [reply src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx:168]: 😄

File: src/applications/personalization/profile/components/account-security/AccountSecurityContent.jsx (hunk 1 above)
Comment (sanlouise) [reply src/applications/personalization/profile/components/account-security/AccountSecurityContent.jsx]: The reason for these overrides is mostly the spacing between the `a` tags, which without an explicit margin will render awkwardly close to one another. So I definitely need to keep those ones 🙂 

I agree that it's generally better practice to not override `h` tag styles like this, I removed the added styling and the screenshot below reflects how it looks now

image

**Pull request merged**